cli.md

```diff
@@ -9,6 +9,7 @@ To guarantee a consistent user experience when using codemodder codemods, we off
 | --output          | the output file to produce (required) |
 | --output-format   | the format for the data output file (codetf or diff) |
 | --sarif           | comma-separated set of path(s) to SARIF file(s) to feed to the codemods |
+| --sonar-issues-json-file | the path to a file containing output from Sonar's Issues API |
```

Comment:
I'm not going to pick a fight over it but I'd be happy with just `--sonar-json` for a slightly shorter name.

Reply:
I guess I wanted to leave room for other Sonar things, or other Sonar formats, etc.

Reply:
Maybe just `--sonar-issues-json` then? Since we don't have `--sarif-file`? Also we should add `--bikeshed` 😅

Reply:
Done